src/components/Cautions/Cautions.js

```diff
@@ -30,10 +30,10 @@ const Cautions = () => (
       source={alert}
   />
     <View style={styles.list}>
-      <CautionItem text={I18n.t('Cautions.keepitSafe')} />
-      <CautionItem text={I18n.t('Cautions.makeBackup')} />
-      <CautionItem text={I18n.t('Cautions.dontShare')} />
-      <CautionItem text={I18n.t('Cautions.dontLose')} />
+      <CautionItem text={textConstants.Cautions.keepItSafe} />
```

Comment:
This file looks like story for StoryBook, not like a component for production.
I see that it was done in develop. But it looks wrong. I suggest to remove `Cautions.js` and all related files/stories.

Reply:
Or I could rewrite it to show 'Caution' according to received param?

Reply:
I'm sure that we just do not need Cautions component. There is no use case when we need all 4 at the same time.